index.js

```diff
@@ -142,8 +203,19 @@ function getAllMoviesReleasedAtOrBeforeYear() {}
  *  getBiggestBoxOfficeMovie(movies);
  *  //> "Incredibles 2"
  */
-function getBiggestBoxOfficeMovie() {}
+function getBiggestBoxOfficeMovie(movies) {
+  if (!movies || !movies.length) {
+   return null;
+}
+let highestBox = movies[0].boxOffice;
+for(let i = 0; i < movies.length; i++){
+  if (movies[i].boxOffice > highestBox) {
+    highestBox = movies[i].title;
+  }
```

Comment:
This is a great set up - you're on the right track.  But lets look at what our `movies[i].boxOffice` looks like:
`$248,757,044` - is an example.

This has some extra characters in it _and its a string_.  
Think about what kind of comparison we need to do.